README.md

```diff
@@ -71,7 +71,7 @@ Here's a tl;dr summary to get the operator and plugin installed in your cluster.
 
 1. First run `make cluster-status` to expose the internal image registry and get the podman command needed to log into the internal image registry.
 2. Run the podman login command that will log into the internal image registry.
-3. Build, push, and deploy the operator and plugin by running `make cluster-push operator-create install-cr`
+3. Build, push, and deploy the operator and plugin by running `make cluster-push operator-create install-crd install-cr`
```

Comment:
Hmm. that's interesting. I thought the operator-create target does the install-crd for you. Now I see why you were confused... I guess I forgot to put that in the docs because I must have thought it did it already.